<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>348</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>348</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>6盘</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(255, 255, 255);
</string>
  </property>
  <widget class="QLabel" name="TitleIcon">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>167</width>
     <height>167</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resource.qrc">:/image/assets/icon_83.5@2x.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="userEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-width:0;
border-style:outset;</string>
   </property>
   <property name="text">
    <string>15618537992</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;border-width:0;border-style:outset</string>
   </property>
   <property name="text">
    <string>woshishei</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>242</y>
     <width>231</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(215,215,215)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>292</y>
     <width>231</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(215,215,215)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QCheckBox" name="autoLogin">
      <property name="styleSheet">
       <string notr="true">color:rgb(215,215,215);</string>
      </property>
      <property name="text">
       <string>自动登录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="regis">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border-width:0;
border-style:outset;
color:rgb(215,215,215);</string>
      </property>
      <property name="text">
       <string>注册账号</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Login">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>450</y>
     <width>257</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(46, 193, 124);
border:none;
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="savePassword">
      <property name="styleSheet">
       <string notr="true">color:rgb(215,215,215);
selection-background-color: rgb(46, 193, 124);
</string>
      </property>
      <property name="text">
       <string>记住密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="forget">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border-width:0;
border-style:outset;
color:rgb(215,215,215);</string>
      </property>
      <property name="text">
       <string>忘记密码</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
